tests/abi/abi.c

```diff
@@ -0,0 +1,100 @@
+#include <stdint.h>
```

Comment:
Copyright notice?

If we don't have it in revng-qa, please add it everywhere (see what revng does for C files).

Reply:
[Done](https://github.com/revng/revng-qa/pull/4/commits/cfb308ce4890fb6cc01c353d65ad41be3c0fa26a).